roles/splunk_monitor/tasks/post_calls.yml

```diff
@@ -25,4 +25,16 @@
     - "{{ dmc_group_kv_store }}"
     - "{{ dmc_group_license_master }}"
     - "{{ dmc_group_search_head }}"
-    - "{{ dmc_group_shc_deployer }}"
\ No newline at end of file
+    - "{{ dmc_group_shc_deployer }}"
+
+- name: Cluster Label POST Requests
+  uri:
+    url: "{{ cert_prefix }}://127.0.0.1:{{ splunk.svc_port }}/services/search/distributed/groups"
+    method: POST
+    user: "{{ splunk.admin_user }}"
+    password: "{{ splunk.password }}"
+    body: "member=localhost:localhost&default=false&name=dmc_indexerclustergroup_{{ item }}"
```

Comment:
This doesn't feel correct to me - `localhost`/`splunk_monitor` here shouldn't be a member of the indexer cluster group? I would expect that the indexers, maybe the cluster master, form the group?

Reply:
updated